app/workers/send_new_cycle_has_started_email_to_candidates_worker.rb

```diff
@@ -0,0 +1,9 @@
+class SendNewCycleHasStartedEmailToCandidatesWorker
+  include Sidekiq::Worker
+
+  def perform
+    GetUnsuccessfulAndUnsubmittedApplicationsFromPreviousCycle.call.each do |application|
```

Comment:
Do you think we need to batch this since it will be in the 10's of thousands of emails sent?